src/cityreader/cityreader.py

```diff
@@ -14,15 +22,26 @@
 #
 # Note that the first line of the CSV is header that describes the fields--this
 # should not be loaded into a City object.
+
 cities = []
 
+
 def cityreader(cities=[]):
-  # TODO Implement the functionality to read from the 'cities.csv' file
-  # For each city record, create a new City instance and add it to the 
-  # `cities` list
-    
+    # TODO Implement the functionality to read from the 'cities.csv' file
+    # For each city record, create a new City instance and add it to the
+    # `cities` list
+    with open('cities.csv', 'r') as city_csv:
+        city_reader = csv.reader(city_csv)
+        line = 0
+        for row in city_reader:
+            if line == 0:
+                line += 1
```

Comment:
Ahh, smart! Avoid the first line, it's just going to be the headers/titles of the columns and such. Well thought out!